vmdb/db/migrate/20101028195038_remove_custom_fields_from_vms_hosts.rb

```diff
@@ -38,7 +38,7 @@ def self.down
   def self.move_custom_vars(klass, custom_names)
     say_with_time("Migrate #{klass.name} custom columns to CustomAttribute") do
       condition = (custom_names).join(' is not NULL OR ') +  ' is not NULL'
-      items = klass.find(:all, :select=>'id,' + custom_names.join(','), :conditions => condition)
+      items = klass.select('id,' + custom_names.join(',')).where(condition)
```

Comment:
Instead of building a string, can we just use the array? `klass.select(custom_names + ["id"]).where(condition)`